app/src/main/java/com/hedvig/app/feature/embark/passages/previousinsurer/PreviousInsurerAdapter.kt

```diff
@@ -0,0 +1,73 @@
+package com.hedvig.app.feature.embark.passages.previousinsurer
+
+import android.graphics.drawable.PictureDrawable
+import android.net.Uri
+import android.view.View
+import android.view.ViewGroup
+import androidx.recyclerview.widget.ListAdapter
+import androidx.recyclerview.widget.RecyclerView
+import com.bumptech.glide.RequestBuilder
+import com.hedvig.app.R
+import com.hedvig.app.databinding.ExpandableBottomSheetTitleBinding
+import com.hedvig.app.databinding.PreviousInsurerItemLayoutBinding
+import com.hedvig.app.util.GenericDiffUtilItemCallback
+import com.hedvig.app.util.extensions.inflate
+import com.hedvig.app.util.extensions.viewBinding
+
+class PreviousInsurerAdapter(
+    previousInsurers: List<PreviousInsurerData.PreviousInsurer>,
+    private val requestBuilder: RequestBuilder<PictureDrawable>,
+    private val onInsurerClicked: (String) -> Unit
+) : ListAdapter<PreviousInsurerItem, PreviousInsurerAdapter.PreviousInsurerViewHolder>(GenericDiffUtilItemCallback()) {
+
+    init {
+        submitList(listOf(PreviousInsurerItem.Header(text = "Insurers")) + previousInsurers.map { it.toListItem() })
```

Comment:
👮 

We should use some form of localizable resource for this title 🙂

Reply:
Ops! How do we usually go about defining new string resources? Add the english version to Lokalise or to strings.xml?

Reply:
Design have usually created the localized resources we need for a feature, let's check it out together on screen share 🙂